src/diff/index.js

```diff
@@ -118,7 +118,7 @@ export function diff(parentDom, newVNode, oldVNode, context, isSvg, excessDomChi
 
 			tmp = c.render(c.props, c.state, c.context);
 			let isTopLevelFragment = tmp != null && tmp.type == Fragment && tmp.key == null;
-			newVNode._children = isTopLevelFragment ? tmp.props.children : tmp;
+			newVNode._children = toChildArray(isTopLevelFragment ? tmp.props.children : tmp);
```

Comment:
nit: should we pass the Array directly? (`toChildArray(.., [])`)

Reply:
Well the third argument of `toChildArray` is actually the returned flattened array, and if that's missing is defined inside the function and returned. Was thinking that without passing it we could save some bytes.